opossum-tool/OpossumUI
# feat: improve display of information in attribution column

### Summary of changes

- add new button components: MenuButton and SplitButton
- MenuButton renders a button which opens a menu with options
- SplitButton renders a button which opens a menu with options and a default action button
- license field: use built-in fullWidth prop
- text fields: properly hide label spans which result in which areas appearing behind text field labels in certain cases
- TextBox: use ellipsis to improve display of overflowing text in input field

### Context and reason for change

Part of #2332 

### How can the changes be tested

Check out license field and text box inputs that they look and work as before.
Since we don't have a [Storybook](https://storybook.js.org/), to see the new buttons in action you either need to look at the next PR (#2358) or render them somewhere locally.
